### PR TITLE
[T] (quests) laserio

### PR DESCRIPTION
**Pull request by AnzhiZhang**
Co-authored-by: Cactusstudent <97865546+Cactusstudent@users.noreply.github.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/laserio.snbt [hunk 1]
```diff
@@ -75,7 +75,7 @@
 		{
 			x: 0.5d
 			y: 0.5d
-			description: ["Used to connect laser connectors and laser nodes to each other"]
+			description: ["用于将laser connectors和laser nodes与其他连接"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/laserio.snbt]: 1

File: config/ftbquests/quests/chapters/laserio.snbt [hunk 2]
```diff
@@ -11,7 +11,7 @@
 		{
 			x: -1.0d
 			y: -1.0d
-			description: ["Used to make logic chips"]
+			description: ["用于制作logic chips"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/laserio.snbt]: 1

File: config/ftbquests/quests/chapters/laserio.snbt [hunk 3]
```diff
@@ -109,7 +109,7 @@
 		{
 			x: 2.0d
 			y: -1.0d
-			description: ["Can be used to extend laser range"]
+			description: ["可用于拓展laser range"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/laserio.snbt]: 1

File: config/ftbquests/quests/chapters/laserio.snbt [hunk 4]
```diff
@@ -161,7 +161,7 @@
 		{
 			x: 5.0d
 			y: 0.5d
-			description: ["Provides basic filtering for items. Can be used on extracting and inserting"]
+			description: ["为物品提供basic filtering。可以用于抽取与输入"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/laserio.snbt]: 1

File: config/ftbquests/quests/chapters/laserio.snbt [hunk 5]
```diff
@@ -126,7 +126,7 @@
 		{
 			x: 5.0d
 			y: -1.0d
-			description: ["Allows the node to move items around"]
+			description: ["允许the node移动物品"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/laserio.snbt]: 1

**Review by AnzhiZhang — COMMENTED** [review]
等模组翻译

File: config/ftbquests/quests/chapters/laserio.snbt (hunk 1 above)
Comment (AnzhiZhang) [reply config/ftbquests/quests/chapters/laserio.snbt]:
```suggestion
			description: ["用于连接镭射连接器和镭射节点"]
```

File: config/ftbquests/quests/chapters/laserio.snbt (hunk 2 above)
Comment (AnzhiZhang) [reply config/ftbquests/quests/chapters/laserio.snbt]:
```suggestion
			description: ["用于制作逻辑芯片"]
```

File: config/ftbquests/quests/chapters/laserio.snbt (hunk 3 above)
Comment (AnzhiZhang) [reply config/ftbquests/quests/chapters/laserio.snbt]:
```suggestion
			description: ["可用于拓展镭射距离"]
```

File: config/ftbquests/quests/chapters/laserio.snbt (hunk 4 above)
Comment (AnzhiZhang) [reply config/ftbquests/quests/chapters/laserio.snbt]:
```suggestion
			description: ["为物品提供基础过滤，可以用于抽取与输入"]
```

File: config/ftbquests/quests/chapters/laserio.snbt (hunk 5 above)
Comment (AnzhiZhang) [reply config/ftbquests/quests/chapters/laserio.snbt]:
```suggestion
			description: ["允许节点移动物品"]
```

File: config/ftbquests/quests/chapters/laserio.snbt [hunk 6]
```diff
@@ -308,7 +308,7 @@
 		{
 			x: 5.0d
 			y: -2.5d
-			description: ["Make card work even faster and transfer more items at once"]
+			description: ["让card工作得更快，一次传输更多物品"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/laserio.snbt]:
```suggestion
			description: ["让卡片工作得更快，一次传输更多物品"]
```

**Review by Cactusstudent — APPROVED** [review]

**Pull request merged**